### ISSUE TITLE
cannot convert freezed model with nchw commmandline input

### ISSUE THREAD

**Issue by shubham-shahh** [onnx/tensorflow-onnx#1334]
Hi I am using the following command to convert my frozen model to onnx model

`python3 -m tf2onnx.convert --input ./facenet_freezed.pb --inputs input_1:0[1,160,160,3] --inputs-as-nchw input_1:0[1,3,160,160] --outputs Bottleneck_BatchNorm/batchnorm_1/add_1:0 --output facenet.onnx`

still my model's input name is [1,160,160,3]. I think am using the above command incorrectly, can someone please correct me?

**Comment by TomWildenhain-Microsoft** [comment]
Try:

`python3 -m tf2onnx.convert --input ./facenet_freezed.pb --inputs input_1:0[1,160,160,3] --inputs-as-nchw input_1:0 --outputs Bottleneck_BatchNorm/batchnorm_1/add_1:0 --output facenet.onnx`

The shape hint should only be provided to the --inputs flag.

**Comment by shubham-shahh** [comment]
> Try:
> 
> `python3 -m tf2onnx.convert --input ./facenet_freezed.pb --inputs input_1:0[1,160,160,3] --inputs-as-nchw input_1:0 --outputs Bottleneck_BatchNorm/batchnorm_1/add_1:0 --output facenet.onnx`
> 
> The shape hint should only be provided to the --inputs flag.

This works, thanks.
